### PR TITLE
Revamped localization system (much more accurate and up to date)

### PR DESCRIPTION
**Pull request by Wind-Explorer**
Updated UI with more accurate and up-to-date sentences for much better clarity
Wiped all translations since most are outdated and irrelevant in the current code and it is a good idea to get everything translated again (a good chance for more people to contribute)

with the new key system for strings, it is now much more easier to manage them and not worrying about typos for synchronisation between source code and locale files

A continuation from PR #84 due to some ridiculous code merging that caused my previous work to go kaboom 💀

### CODE REVIEW

File: PlayCover/View/AppSettingsView.swift [hunk 1]
```diff
@@ -27,64 +27,62 @@ struct AppSettingsView: View {
...
             Divider().padding(.leading, 36).padding(.trailing, 36)
             HStack {
                 Spacer()
-                Button("Reset settings and keymapping") {
+                Button("settings.reset") {
                     resetedAlert.toggle()
                     settings.reset()
-                }.buttonStyle(GrowingButton()).padding()
-                Button("Ok") {
+                }.buttonStyle(CancelButton()).padding()
```
Comment (IsaacMarovitz) [inline PlayCover/View/AppSettingsView.swift:84]: Why did you change the button style here?

File: PlayCover/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,44 +1,90 @@
...
-"It is impossible to decrypt .IPA with SIP disabled. Please, enable it." = "Es ist unmöglich, .IPA mit deaktiviertem SIP zu entschlüsseln. Bitte aktiviere es.";
-"Please reinstall PlayCoverApp" = "Bitte installiere die PlayCoverApp neu";
-"Make sure you don't disallowed PlayCover to access files in Settings - Secuirity & Privacy" = "Stelle sicher, dass Du PlayCover den Zugriff auf Dateien in den Einstellungen – Sicherheit und Datenschutz nicht untersagt hast";
-"This .IPA is courrupted.Try to use another .ipa" = "Diese .IPA ist beschädigt.Versuch, eine andere .ipa . zu verwenden";
-"You have not performed all instructions! Please, perform all instructions in 'Settings' or watch video in pinned messages." = "Du hast nicht alle Anweisungen ausgeführt! Bitte führe alle Anweisungen unter 'Einstellungen' aus oder schau Dir das Video in den angepinnten Nachrichten an..";
-"Get" = "Herunterladen";
-"soundAlert.messageText" = "";
-"soundAlert.informativeText" = "";
-"soundAlert.successText" = "";
+"app.add" = "Add app";
```
Comment (IsaacMarovitz) [inline PlayCover/de.lproj/Localizable.strings:1]: Would it not be better to salvage the localisations that are already present?

File: PlayCover/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -1,44 +1,90 @@
...
-"This .IPA is corrupted. It doesn't contain Info.plist." = "Diese .IPA ist beschädigt. Sie enthält keine Info.plist.";
-"It is impossible to decrypt .IPA with SIP disabled. Please, enable it." = "Es ist unmöglich, .IPA mit deaktiviertem SIP zu entschlüsseln. Bitte aktiviere es.";
-"Please reinstall PlayCoverApp" = "Bitte installiere die PlayCoverApp neu";
-"Make sure you don't disallowed PlayCover to access files in Settings - Secuirity & Privacy" = "Stelle sicher, dass Du PlayCover den Zugriff auf Dateien in den Einstellungen – Sicherheit und Datenschutz nicht untersagt hast";
-"This .IPA is courrupted.Try to use another .ipa" = "Diese .IPA ist beschädigt.Versuch, eine andere .ipa . zu verwenden";
-"You have not performed all instructions! Please, perform all instructions in 'Settings' or watch video in pinned messages." = "Du hast nicht alle Anweisungen ausgeführt! Bitte führe alle Anweisungen unter 'Einstellungen' aus oder schau Dir das Video in den angepinnten Nachrichten an..";
-"Get" = "Herunterladen";
-"soundAlert.messageText" = "";
```
Comment (IsaacMarovitz) [inline PlayCover/de.lproj/Localizable.strings:42]: You're removing the localisations for `soundAlert.*`

**Review by IsaacMarovitz — CHANGES_REQUESTED** [review]

File: PlayCover/de.lproj/Localizable.strings (hunk 2 above)
Comment (Wind-Explorer) [reply PlayCover/de.lproj/Localizable.strings:1]: Discussion existed before [here](https://ptb.discord.com/channels/871829896492642387/996713647827066981/999956158456537158)

**Review by JoseMoreville — COMMENTED** [review]
Seems correct

File: PlayCover/de.lproj/Localizable.strings [hunk 4]
```diff
@@ -1,44 +1,93 @@
...
+"app.importkm" = "Import keymapping";
+"app.exportkm" = "Export keymapping";
+"app.delete" = "App löschen";//
+"alert.appCacheCleared" = "App data has been cleared!";
+"alert.kmImported" = "Keymapping imported!";
+"alert.errorImportKm" = "Error occured when importing!";
+"alert.app.delete" = "All app data will be erased. You may need to download app files again. Do you wish to continue?";
+"alert.wrongFileType" = "Falscher Dateityp";//
+"button.OK" = "OK";
+"button.Proceed" = "Proceed";
```
Comment (IsaacMarovitz) [inline PlayCover/de.lproj/Localizable.strings:16]: Proceed sounds too formal. 'Continue' would be better

File: PlayCover/de.lproj/Localizable.strings [hunk 5]
```diff
@@ -1,44 +1,93 @@
...
+"menubar.downloadMoreApps" = "Download more apps";
+"menubar.downloadMoreApps.help" = "Use this site to download apps for installation";
+"menubar.exportToSideloady" = "Export to Sideloadly";
+"menubar.exportToSideloady.help" = "Export IPAs for Sideloady from sideloady.io which does not need SIP to be disabled for apps and games to work.";
+"alert.wrongFileType.message" = "Choose an .ipa file";
+"app.add.help" = "Drag or open an app file to install. IPAs from Configurator or iMazing won't work! You should get decrypted IPAs, either from the top right button, Discord, AppDb, or a jailbroken device.";
+"app.update" = "Update app";
+"bottomBar.notices" = "Notices";
+"alert.followInstructionAndRestartApp" = "Please follow the given instructions, and restart the App.";
+"xcode.installFailed" = "Xcode Command Line Tools intallation failed";
```
Comment (IsaacMarovitz) [inline PlayCover/de.lproj/Localizable.strings:32]: This should be with the other Xcode related stuff lower down

File: PlayCover/de.lproj/Localizable.strings [hunk 6]
```diff
@@ -1,44 +1,93 @@
...
+"debug.crash" = "Crash app";
+"menubar.log.copy" = "Copy Logs";
+"menubar.documentation" = "Documentation";
+"menubar.website" = "Website";
+"menubar.github" = "GitHub";
+"menubar.discord" = "Discord";
+"menubar.showAppLinks" = "Show app links";
+"xcode.needToInstall" = "You need to install Xcode Commandline tools and restart this App.";
+"button.Install" = "Install";
+"alert.xcodeToolsInstallSucceeded" = "Xcode tools installation succeeded";
```
Comment (IsaacMarovitz) [inline PlayCover/de.lproj/Localizable.strings:89]: `alert.xcodeToolsInstallSucceeded` is not consistent with other Xcode-based localisations.

File: PlayCover/de.lproj/Localizable.strings [hunk 7]
```diff
@@ -1,44 +1,93 @@
...
+"button.Cancel" = "Cancel";
+"button.Close" = "Close";
+"button.Dismiss" = "Dismiss";
+"error.corruptedIPA" = "Diese .IPA ist beschädigt. Sie enthält keine Info.plist.";//
+"error.waitInstallation" = "Please wait for the current installation to finish!";
+"error.reinstallApp" = "Bitte installiere die PlayCover neu";
+"menubar.downloadMoreApps" = "Download more apps";
+"menubar.downloadMoreApps.help" = "Use this site to download apps for installation";
+"menubar.exportToSideloady" = "Export to Sideloadly";
+"menubar.exportToSideloady.help" = "Export IPAs for Sideloady from sideloady.io which does not need SIP to be disabled for apps and games to work.";
```
Comment (IsaacMarovitz) [inline PlayCover/de.lproj/Localizable.strings:26]: We need consistency when referring to `.ipa` files. In this localisation alone, there's ".IPA", "IPA", ".ipa", and "ipa". IMO they should all be '.ipa'.

File: PlayCover/de.lproj/Localizable.strings [hunk 8]
```diff
@@ -1,44 +1,93 @@
...
+"adaptiveDisplay" = "Adaptive display";
+"adaptiveDisplay.info" ="Adapts and scales app window to fit the built-in Mac display.";
+"jbBypass.enable" = "Enable Jailbreak Bypass (Alpha)";
+"jbBypass.enable.info" = "Attempts to bypass Jailbreak Detection in some games (No guarentee to work all the time)";
+"displayRefreshRate" = "Screen refresh rate";
+"mouseSensitivity" = "Mouse sensitivity: ";
+"settings.reset" = "Reset settings and keymapping";
+"settings.resetted" = "Settings have been reverted back to default!";
+"setupView.header" = "Are apps not allowing you to login?";
+"setupView.subheader" = "Follow the instructions below to fix exactly that!";
```
Comment (IsaacMarovitz) [inline PlayCover/de.lproj/Localizable.strings:49]: This is too casual and colloquial to localise nicely.

File: PlayCover/en.lproj/Localizable.strings [hunk 9]
```diff
@@ -1,114 +1,93 @@
...
+"menubar.log.copy" = "Copy Logs";
+"menubar.documentation" = "Documentation";
+"menubar.website" = "Website";
+"menubar.github" = "GitHub";
+"menubar.discord" = "Discord";
+"menubar.showAppLinks" = "Show app links";
+"xcode.needToInstall" = "You need to install Xcode Commandline tools and restart this App.";
+"button.Install" = "Install";
+"alert.xcodeToolsInstallSucceeded" = "Xcode tools installation succeeded";
+"failedToFetchNotices" = "Failed to find notices. Are we not online?";
```
Comment (IsaacMarovitz) [inline PlayCover/en.lproj/Localizable.strings:90]: 'Are we not online?' is too casual should be 'Please check your internet connection' or something.

**Review by IsaacMarovitz — CHANGES_REQUESTED** [review]

File: PlayCover/de.lproj/Localizable.strings [hunk 10]
```diff
@@ -1,44 +1,93 @@
...
+"error.reinstallApp" = "Bitte installiere die PlayCover neu";
+"menubar.downloadMoreApps" = "Download more apps";
+"menubar.downloadMoreApps.help" = "Use this site to download apps for installation";
+"menubar.exportToSideloady" = "Export to Sideloadly";
+"menubar.exportToSideloady.help" = "Export IPAs for Sideloady from sideloady.io which does not need SIP to be disabled for apps and games to work.";
+"alert.wrongFileType.message" = "Choose an .ipa file";
+"app.add.help" = "Drag or open an app file to install. IPAs from Configurator or iMazing won't work! You should get decrypted IPAs, either from the top right button, Discord, AppDb, or a jailbroken device.";
+"app.update" = "Update app";
+"bottomBar.notices" = "Notices";
+"alert.followInstructionAndRestartApp" = "Please follow the given instructions, and restart the App.";
```
Comment (IsaacMarovitz) [inline PlayCover/de.lproj/Localizable.strings:31]: Please consolidate all `alert.*` locals in one place. They're too scattered around rn.

File: PlayCover/de.lproj/Localizable.strings (hunk 4 above)
Comment (Wind-Explorer) [reply PlayCover/de.lproj/Localizable.strings:16]: The button carries out actions that are irreversible and may cause unwanted data loss, and hence I would still prefer "Proceed" over "Continue" as it sounds more serious and that it conveys the same idea to the user

**Comment by IsaacMarovitz** [comment]
Reviewing one sec

File: PlayCover/View/AppsView.swift [hunk 11]
```diff
@@ -28,26 +25,24 @@ struct AppsView: View {
...
-					Button("Install") {
+					Button("button.Install") {
 						do {
 							_ = try shell.sh("xcode-select --install")
-							alertTitle = NSLocalizedString("Xcode tools installation succeeded", comment: "")
-							alertBtn = NSLocalizedString("Close", comment: "")
-							alertText = NSLocalizedString("Please follow the given instructions, and restart the App.", comment: "")
+							alertTitle = NSLocalizedString("xcode.xcodeToolsInstallSucceeded", comment: "")
+							alertBtn = NSLocalizedString("button.Close", comment: "")
+							alertText = NSLocalizedString("alert.followInstructionAndRestartApp", comment: "")
```
Comment (IsaacMarovitz) [inline PlayCover/View/AppsView.swift:38]: `Instruction` should be plural IMO.

File: PlayCover/View/MainView.swift [hunk 12]
```diff
@@ -19,16 +17,17 @@ struct SearchView: View {
 
     @State private var search: String = ""
     @State private var isEditing = false
+    @State private var darkSearchStroke = Color(red: 0.2, green: 0.2, blue: 0.2)
```
Comment (IsaacMarovitz) [inline PlayCover/View/MainView.swift:20]: These variables do not need to be `@State`

File: PlayCover/View/MainView.swift [hunk 13]
```diff
@@ -67,7 +71,7 @@ struct MainView: View {
     @State var showSetup = false
     @State var noticesExpanded = false
     @State var bottomHeight: CGFloat = 0
-
+    @State private var redColor = Color(red: 0.9, green: 0, blue: 0)
```
Comment (IsaacMarovitz) [inline PlayCover/View/MainView.swift:74]: Also does not need to be `@State`

File: PlayCover/View/PlayAppView.swift [hunk 14]
```diff
@@ -88,21 +87,21 @@ struct PlayAppView: View {
                         }
                     }
                 }, label: {
-                    Text("Import keymapping")
+                    Text("app.importkm")
```
Comment (IsaacMarovitz) [inline PlayCover/View/PlayAppView.swift:90]: Should be camelCase `app.importKm`

File: PlayCover/View/PlayAppView.swift [hunk 15]
```diff
@@ -88,21 +87,21 @@ struct PlayAppView: View {
...
                     Image(systemName: "square.and.arrow.down.on.square.fill")
                 })
 
                 Button(action: {
                     app.settings.export()
                 }, label: {
-                    Text("Export keymapping")
+                    Text("app.exportkm")
```
Comment (IsaacMarovitz) [inline PlayCover/View/PlayAppView.swift:97]: Should be camelCase `app.exportKm`

**Review by IsaacMarovitz — CHANGES_REQUESTED** [review]

File: PlayCover/de.lproj/Localizable.strings [hunk 16]
```diff
@@ -1,44 +1,133 @@
...
-"It is impossible to decrypt .IPA with SIP disabled. Please, enable it." = "Es ist unmöglich, .IPA mit deaktiviertem SIP zu entschlüsseln. Bitte aktiviere es.";
-"Please reinstall PlayCoverApp" = "Bitte installiere die PlayCoverApp neu";
-"Make sure you don't disallowed PlayCover to access files in Settings - Secuirity & Privacy" = "Stelle sicher, dass Du PlayCover den Zugriff auf Dateien in den Einstellungen – Sicherheit und Datenschutz nicht untersagt hast";
-"This .IPA is courrupted.Try to use another .ipa" = "Diese .IPA ist beschädigt.Versuch, eine andere .ipa . zu verwenden";
-"You have not performed all instructions! Please, perform all instructions in 'Settings' or watch video in pinned messages." = "Du hast nicht alle Anweisungen ausgeführt! Bitte führe alle Anweisungen unter 'Einstellungen' aus oder schau Dir das Video in den angepinnten Nachrichten an..";
-"Get" = "Herunterladen";
-"soundAlert.messageText" = "";
-"soundAlert.informativeText" = "";
-"soundAlert.successText" = "";
+"app.add" = "Add app";
```
Comment (Candygoblen123) [inline PlayCover/de.lproj/Localizable.strings:1]: could we rename the `app` to `playapp`, for clarity.

File: PlayCover/de.lproj/Localizable.strings [hunk 17]
```diff
@@ -1,44 +1,133 @@
...
+
+
+"button.OK" = "OK";
+"button.Proceed" = "Proceed";
+"button.Cancel" = "Cancel";
+"button.Close" = "Close";
+"button.Dismiss" = "Dismiss";
+"button.Install" = "Install";
+
+"error.corruptedIPA" = "This .ipa file is broken. It does not contains Info.plist.";
```
Comment (Candygoblen123) [inline PlayCover/de.lproj/Localizable.strings:58]: "This .ipa file is corrupted. It does not contain an "Info.plist" file.

File: PlayCover/de.lproj/Localizable.strings [hunk 18]
```diff
@@ -1,44 +1,133 @@
...
+
+"bottomBar.notices" = "Notices";
+"bottomBar.notices.help" = "News and announcements from the developers";
+"bottomBar.setupViewButton" = "Experiencing issues?";
+
+"failedToFetchNotices" = "Failed to find notices. Please check your internet connection.";
+
+"logs.copied" = "Logs copied";
+
+"enable.km" = "Enable keymapping";
```
Comment (Candygoblen123) [inline PlayCover/de.lproj/Localizable.strings:75]: rename `enable` to be more descriptive of where these string keys are used

File: PlayCover/de.lproj/Localizable.strings [hunk 19]
```diff
@@ -1,44 +1,133 @@
...
+"enable.gaming" = "Gaming Mode";
+"enable.km.info" = "Add keyboard support to touch-only games. Use Command + K to open editor menu.";
+"enable.adaptiveDisplay" = "Adaptive display";
+"enable.adaptiveDisplay.info" ="Adapts and scales app window to fit the built-in Mac display.";
+"enable.jbBypass" = "Enable Jailbreak Bypass (Alpha)";
+"enable.jbBypass.info" = "Attempts to bypass Jailbreak Detection in some games (No guarentee to work all the time)";
+"enable.autoWindowResize" = "Enable auto window resize";
+
+"displayRefreshRate" = "Screen refresh rate";
```
Comment (Candygoblen123) [inline PlayCover/de.lproj/Localizable.strings:84]: add a prefix to describe where these keys are used

File: PlayCover/de.lproj/Localizable.strings [hunk 20]
```diff
@@ -1,44 +1,133 @@
...
+"mouseSensitivity" = "Mouse sensitivity: ";
+
+"settings.reset" = "Reset settings and keymapping";
+"settings.resetCompleted" = "Settings have been reverted back to default!";
+
+"setupView.header" = "Follow steps below for fixing common issues";
+"setupView.subheader" = "Resolves apps not allowing logins, crashes at start and more";
+
+"setupView.1" = "1) Shutdown your Mac";
```
Comment (Candygoblen123) [inline PlayCover/de.lproj/Localizable.strings:93]: Instead of using just `1`, use `step1`

File: PlayCover/de.lproj/Localizable.strings [hunk 21]
```diff
@@ -1,44 +1,133 @@
...
+
+"setupView.vidTutorials" = "Video Tutorials";
+"setupView.youtube" = "YouTube";
+"setupView.bilibili" = "BiliBili";
+
+"setupView.pressButtonAndEnterPassword" = "Please disable SIP from below and enter your Mac login password.";
+"setupView.pressButtonAndEnterPassword.button" = "Disable SIP";
+
+"notification.appInstalled" = "App installed!";
+"notification.appInstalled.message" = "Check it out now!";
```
Comment (Candygoblen123) [inline PlayCover/de.lproj/Localizable.strings:111]: maybe say to check it out in PlayCover instead

File: PlayCover/de.lproj/Localizable.strings [hunk 22]
```diff
@@ -1,44 +1,133 @@
...
+"menubar.website" = "Website";
+"menubar.github" = "GitHub";
+"menubar.discord" = "Discord";
+"menubar.showAppLinks" = "Show app links";
+
+"soundAlert.messageText" = "Incorrect Audio Settings Detected!";
+"soundAlert.informativeText" = "Your current output device does not have a sample rate of 48 or 44.1 KHz! Crashes may occur. Would you like to change your current output device's sample rate to 48 KHz?";
+"soundAlert.successText" = "Current output device sample rate set to 48 KHz";
+
+"xcode.needToInstall" = "You need to install Xcode Commandline tools and restart this App.";
```
Comment (Candygoblen123) [inline PlayCover/de.lproj/Localizable.strings:130]: maybe use `xcode.install`, `xcode.install.success`, and `xcode.install.fail`

**Review by Candygoblen123 — CHANGES_REQUESTED** [review]

File: PlayCover/View/MainView.swift [hunk 23]
```diff
@@ -66,7 +70,7 @@ struct MainView: View {
...
```
Comment (IsaacMarovitz) [inline PlayCover/View/MainView.swift:73]: Make not `@State` please then I can approve :)

Comment (IsaacMarovitz) [inline PlayCover/View/MainView.swift:73]: Please make not `@State` then I can approve :)

**Review by IsaacMarovitz — CHANGES_REQUESTED** [review]

File: PlayCover/View/AppSettingsView.swift [hunk 24]
```diff
@@ -87,18 +83,22 @@ struct AppSettingsView: View {
...
                 Spacer()
-                Button("Reset settings and keymapping") {
-                    resetedAlert.toggle()
+                Button("settings.reset") {
+                    resetCompletedAlert.toggle()
                     settings.reset()
-                }.buttonStyle(GrowingButton()).padding()
-                Button("Ok") {
+                }
+                .padding([.top, .bottom])
```
Comment (Depal1) [inline PlayCover/View/AppSettingsView.swift:97]: UI is for another PR. Please remove and open another PR for these changes.

**Review by Depal1 — CHANGES_REQUESTED** [review]

**Review by IsaacMarovitz — APPROVED** [review]

**Comment by Depal1** [comment]
There are conflicts on this PR, a rebase is required.

File: PlayCover/View/DeleteGenshinStoredAccountView.swift [hunk 25]
```diff
@@ -27,15 +27,19 @@ struct DeleteGenshinStoredAccountView: View {
                             Image(systemName: "person.fill")
                             Text(account)
                         }.frame(minWidth: 300, alignment: .center)
-                    }.controlSize(.large).buttonStyle(GrowingButton()).font(.title3)
-                        .frame(width: 300, alignment: .center)
+                    }
```
Comment (Depal1) [inline PlayCover/View/DeleteGenshinStoredAccountView.swift:30]: UI changes don't belong to this PR. Please open another PR for those

File: PlayCover/View/MainView.swift [hunk 26]
```diff
@@ -40,20 +39,25 @@ struct SearchView: View {
...
-                        .frame(minWidth: 0, maxWidth: .infinity, alignment: .leading)
-                        .padding(.leading, 16)
-                    if isEditing {
-                            Button(action: {
-                            self.search = ""
-                            }, label: {
-                            Image(systemName: "multiply.circle.fill")
-                                .padding(.trailing, 16)
-                        }).buttonStyle(PlainButtonStyle())
+                ZStack {
```
Comment (Depal1) [inline PlayCover/View/MainView.swift:42]: More UI changes that don't belong on this PR.

File: PlayCover/View/MainView.swift [hunk 27]
```diff
@@ -91,17 +95,21 @@ struct MainView: View {
...
-                                Text("Notices").font(.headline).help("Important news and announcements")
+                                Text("bottomBar.notices").font(.headline).help("bottomBar.notices.help")
                                 Button {
                                     withAnimation { noticesExpanded.toggle() }
                                 } label: {
                                     Image(systemName: "chevron.up")
                                         .rotationEffect(Angle(degrees: noticesExpanded ? 180 : 0))
                                 }
                                 Spacer()
+                                #if DEBUG
```
Comment (Depal1) [inline PlayCover/View/MainView.swift:106]: The same here. UI changes don't belong to this PR.

File: PlayCover/View/MainView.swift [hunk 28]
```diff
@@ -117,14 +125,9 @@ struct MainView: View {
...
                         }.frame(maxWidth: .infinity)
-						#if DEBUG
-						Divider()
-						HStack(spacing: 12) {
-							Button("Crash") { fatalError("Crash was triggered") }
-								.buttonStyle(.borderedProminent).tint(.accentColor).controlSize(.large)
-						}.frame(maxWidth: .infinity)
-						#endif
-                    }.padding()
+                    }
```
Comment (Depal1) [inline PlayCover/View/MainView.swift:128]: UI changes don't belong to this PR.

File: PlayCover/View/StoreGenshinAccountView.swift [hunk 29]
```diff
@@ -53,14 +53,18 @@ struct StoreGenshinAccountView: View {
                 } else { presentationMode.wrappedValue.dismiss()
                 }
             }) {
-                Text("Store!").frame(minWidth: 300, alignment: .center)
-                }.controlSize(.large).buttonStyle(GrowingButton()).font(.title3)
+                Text("storeAccount.store").frame(minWidth: 300, alignment: .center)
```
Comment (Depal1) [inline PlayCover/View/StoreGenshinAccountView.swift:56]: UI changes don't belong to this PR. Please open another PR for UI changes.

**Review by Depal1 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 30]
```diff
@@ -97,3 +97,4 @@ fastlane/test_output
 iOSInjectionProject/
 xcuserdata/*
 PlayCover.xcodeproj/project.pbxproj
+/PlayCover.xcodeproj/project.pbxproj
```
Comment (IsaacMarovitz) [inline .gitignore:100]: Please PLEASE **PLEASE** NEVER DO THIS lmao. Add back project.pbxproj

**Review by IsaacMarovitz — CHANGES_REQUESTED** [review]

File: PlayCover/View/MainView.swift [hunk 31]
```diff
@@ -22,13 +20,12 @@ struct SearchView: View {
...
     var body : some View {
-        TextField(NSLocalizedString("Search...", comment: ""), text: $search)
+        TextField(NSLocalizedString("search.search", comment: ""), text: $search)
             .padding(7)
             .padding(.horizontal, 25)
             .background(Color(NSColor.textBackgroundColor))
-            .cornerRadius(8)
             .font(Font.system(size: 16))
-            .padding(.horizontal, 10)
+            .cornerRadius(8)
```
Comment (IsaacMarovitz) [inline PlayCover/View/MainView.swift:28]: UI Change

File: PlayCover/View/MainView.swift [hunk 32]
```diff
@@ -54,6 +51,7 @@ struct SearchView: View {
                     }
                 }
             )
+            .padding(.horizontal, 10)
```
Comment (IsaacMarovitz) [inline PlayCover/View/MainView.swift:54]: UI Change

File: PlayCover/View/MainView.swift [hunk 33]
```diff
@@ -66,7 +64,7 @@ struct MainView: View {
     @State var showSetup = false
     @State var noticesExpanded = false
     @State var bottomHeight: CGFloat = 0
-
+    var redColor = Color(red: 0.9, green: 0, blue: 0)
```
Comment (IsaacMarovitz) [inline PlayCover/View/MainView.swift:67]: Remove now unused variable

**Review by IsaacMarovitz — CHANGES_REQUESTED** [review]

File: PlayCover/View/MainView.swift [hunk 34]
```diff
@@ -22,11 +20,10 @@ struct SearchView: View {
     @Environment(\.colorScheme) var colorScheme
 
     var body : some View {
-        TextField(NSLocalizedString("Search...", comment: ""), text: $search)
+        TextField(NSLocalizedString("search.search", comment: ""), text: $search)
             .padding(7)
             .padding(.horizontal, 25)
             .background(Color(NSColor.textBackgroundColor))
-            .cornerRadius(8)
```
Comment (IsaacMarovitz) [inline PlayCover/View/MainView.swift:29]: UI Change

**Review by IsaacMarovitz — CHANGES_REQUESTED** [review]

**Comment by IsaacMarovitz** [comment]
I think that;'s the last remaining UI change. Will need to be rebased and expanded when #120 gets merged.

**Review by Depal1 — APPROVED** [review]
